### PR TITLE
Update types

### PR DESCRIPTION
**Pull request by polemius**
As merge request https://github.com/thoov/mock-socket/pull/254 has been closed.

I've updated the types.

### CODE REVIEW

**Comment by romgain** [comment]
Hi @polemius !

Thank you very much for your contribution, it's great!

There's just one little type error that needs attention, but I'm looking forward to merging this!

File: src/websocket.ts [hunk 1]
```diff
@@ -57,9 +54,8 @@ export default class WS {
 
   on(
     eventName: "connection" | "message" | "close",
-    callback: (socket: Server) => void
+    callback: (socket: WebSocket) => void
```
Comment (romgain) [inline src/websocket.ts]: I think that `Server` was the correct type? This change seems to be breaking the type checking here: https://github.com/romgain/jest-websocket-mock/blob/master/src/__tests__/websocket.test.ts#L145

See the typescript logs in Travis here: https://travis-ci.org/romgain/jest-websocket-mock/jobs/630849316#L261

Comment (polemius) [reply src/websocket.ts]: Thank you for review. In the latest I've reverted `ts-ignore` for this function. Do you think it is good?

Comment (romgain) [reply src/websocket.ts]: Thanks! 

Unfortunately, I think that's the best we can do for now 🙂

**Comment by romgain** [comment]
Thank you @polemius !

It's amazing to see PRs that improve code quality like this! 🙂

**Comment by polemius** [comment]
@romgain Thank you for this very helpful library 🎉

**Pull request merged**